code/modules/events/blob.dm

```diff
@@ -23,6 +23,5 @@
 	var/mob/M = pick(candidates)
 	B.key = M.key
 	to_chat(B, "<span class='userdanger'>You are now a mouse, infected with blob spores. Find somewhere isolated... before you burst and become the blob! Use ventcrawl (alt-click on vents) to move around.</span>")
-	var/image/alert_overlay = image('icons/mob/blob.dmi', "blank_blob")
-	notify_ghosts("Infected Mouse has appeared in [get_area(B)].", source = B, alert_overlay = alert_overlay)
+	notify_ghosts("Infected Mouse has appeared in [get_area(B)].", source = B)

```

Comment:
(What's the reason for this change?)

Reply:
The legion megafauna ghost notification had no icon and Kyet mentioned the mouse blob had the same issue. When I fixed the former I figured I’d include a fix for the blob mouse here.